src/coverage-system/sectionfinder.ts

```diff
@@ -116,7 +116,7 @@ export class SectionFinder {
         const editorFileAbs = normalizeFileName(fileName);
         const workspaceFile = normalizeFileName(workspaceFsPath);
         const editorFileRelative = editorFileAbs.substring(workspaceFile.length);
-        const workspaceFolderName = normalizeFileName(workspaceFolder.name);
+        const workspaceFolderName = normalizeFileName(basename(workspaceFsPath));
```

Comment:
Looking pretty good, I can test on windows once this is in master (and the integration tests pass so probably not an issue). I did see that the windows usage might be different (1) but hopefully the integration tests catch this.

1: https://nodejs.org/api/path.html#path_path_basename_path_ext

Reply:
Hey, apologies for not getting back to you on this. This shouldn't be an issue since the case-sensitivity only affects file extensions from what I can tell in the documentation. VS Code doesn't support pointing workspace folders to files, so this shouldn't be an issue.

Reply:
No problem on not getting back right away, sounds good on the reasoning as well.